.github/workflows/chroma-integration-test.yml

```diff
@@ -32,6 +33,6 @@ jobs:
       with:
         python-version: ${{ matrix.python }}
     - name: Install test dependencies
-      run: python -m pip install -r requirements.txt && python -m pip install -r requirements_dev.txt
+      run: python -m pip install -r requirements.txt && python -m pip install -r requirements_dev.txt && python -m pip install -r requirements_test.txt
```

Comment:
I'd rather not pollute the dependency tree during testing. Let's add the requirements_test.txt as a separate step and only run it if the matrix entry is set the embedding function test.
 
 See checking matrix context here: https://stackoverflow.com/questions/57946173/github-actions-run-step-on-specific-os

Reply:
Hello @HammadB,  just wanted to follow up here :)